dashboard/test/models/script_test.rb

```diff
@@ -2177,6 +2177,24 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     assert_equal 'required', script.lessons[0].lesson_group.key
   end
 
+  test 'can add description and big questions for lesson group' do
+    l = create :level
+    dsl = <<-SCRIPT
+      lesson_group 'lg-1', display_name: 'Lesson Group'
```

Comment:
Maybe add a second lesson group to this test case to test that descriptions and big questions are correctly associated with their lesson group?